.github/workflows/manually_compile_check_image.yml

```diff
@@ -0,0 +1,75 @@
+name: Build image for compiler check
+
+# This is a manual trigger.
+on: [workflow_dispatch]
+
+jobs:
+  generate-centos-image:
+      # GitHub Actions doesn't have CentOS VM provided
+      runs-on: ubuntu-18.04
+
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # Because "Build and push" step `context` field can't be subdir,
+      # we need to copy files needed by dockerfile to root dir of the project
+      - name: Copy context for docker build
+        run: |
+          cp -r .github/workflows/docker .
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./docker/Dockerfile-compile-check-centos8.1
+          platforms: linux/amd64
+          push: true
+          tags: inclavarecontainers/test:compile-check-centos8.1
+
+  generate-ubuntu-image:
+      # GitHub Actions doesn't have CentOS VM provided
+      runs-on: ubuntu-18.04
+
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      # Because "Build and push" step `context` field can't be subdir,
+      # we need to copy files needed by dockerfile to root dir of the project
+      - name: Copy context for docker build
+        run: |
+          cp -r .github/workflows/docker .
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
```

Comment:
Please pay attention to your spelling, `Buildx` isn't a valid word in English.

Reply:
？？Please google `Docker Buildx`.
Docker Buildx is a CLI plugin that extends the docker command with the full support of the features provided by Moby BuildKit builder toolkit